util/start_test

```diff
@@ -644,7 +645,7 @@ endif
 if ($?CHPL_TEST_UTIL_DIR) then
     set utildir = "$CHPL_TEST_UTIL_DIR"
 else
-    set utildir = "$CHPL_HOME/util"
+    set utildir = `dirname $0`
```

Comment:
release-over-release testing has been a mystery to me as well. It looks like it is setting CHPL_TEST_UTIL_DIR to the same location as start_test, though. The fact that it is setting CHPL_TEST_UTIL_DIR means it won't use the new default anyway, so we should be safe. The new default might mean it does not need to set the env var, though.

See: https://github.com/chapel-lang/chapel/blob/master/util/pastPerformance/testReleasesPerformance#L44-46

In my case, I was running start_test from a clone, but it was picking up the sub_test from a module install, which seemed unfortunate. I think in general we want to use the sub_test that is next to start_test unless the caller specifically wants something else, in which case CHPL_TEST_UTIL_DIR will work.